modules/oc-mirror-creating-image-set-config.adoc

```diff
@@ -15,32 +15,58 @@ You must specify a storage backend in the image set configuration file. This sto
 Do not delete or modify the metadata that is generated by the oc-mirror plug-in. You must use the same storage backend every time you run the oc-mirror plug-in for the same mirror registry.
 ====
 
+.Prerequisites
+
+* You have created a container image registry credentials file. For instructions, see _Configuring credentials that allow images to be mirrored_.
+
 .Procedure
 
-. Create an `ImageSetConfiguration` resource that specifies the necessary configuration details:
+. Use the `oc mirror init` command to create a template for the image set configuration and save it to a file called `imageset-config.yaml`:
++
+[source,terminal]
+----
+$ oc mirror init --registry example.com/example/oc-mirror > imageset-config.yaml <1>
+----
+<1> Replace `example.com/example/oc-mirror` with the location of your registry for the storage backend.
+
+. Edit the file and adjust the settings as necessary:
 +
-.Example `ImageSetConfiguration` file
 [source,yaml]
 ----
-apiVersion: mirror.openshift.io/v1alpha1
 kind: ImageSetConfiguration
-archiveSize: 4 <1>
+apiVersion: mirror.openshift.io/v1alpha2
+archiveSize: 4                                                      <1>
+storageConfig:                                                      <2>
+  registry:
+    imageURL: example.com/example/oc-mirror                         <3>
+    skipTLS: false
 mirror:
- ocp:
-   channels:
-     - name: stable-4.9 <2>
- operators:
-   - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.9 <3>
-storageConfig: <4>
- registry:
-   imageURL: example.com/example/oc-mirror <5>
+  platform:
+    channels:
+    - name: stable-0.2                                              <4>
```

Comment:
@jpower432 When I tested this and generated the initial template, this channel is stable-0.2. Shouldn't it default to something for the current OCP version, like `stable-4.11`? (For the 4.10 docs, it was stable-4.9)

I updated this procedure so that the first step is to use `oc mirror init` as a starting point. This is the only field that doesn't look like a good enough starting/default value - wasn't sure if that could be fixed?

Reply:
Okay thanks, I'll go ahead and change it manually to stable-4.11 then.